packages/labs/src/inputList.tsx

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+
+import { IProps, Keys, Utils } from "@blueprintjs/core";
+
+export interface IListItemsProps<T> extends IProps {
+    /** Array of items in the list. */
+    items: T[];
+
+    /** Customize filtering of individual items. Return `true` to keep the item, `false` to hide. */
+    itemPredicate?: (item: T, query: string) => boolean;
+
+    /**
+     * Customize filtering of entire items array. Return subset of items that pass filter.
+     * (Some filter algorithms operate on the entire set, rather than individual items.)
+     */
```

Comment:
Can we clarify how these two props (itemPredicate and itemListPredicate) work together if they're both defined? In that case, is an error thrown, is one of these props ignored, etc. (I see from the code that itemListPredicate takes precedence.)